docs/indexing.md

```diff
@@ -75,27 +102,86 @@ in `/tmp/updates/` and as they are processed they will be moved to `/data/marc_l
 
 Once the files are all downloaded and processed, index them with
 
-`$ SET_URL=http://lib-solr8-staging.princeton.edu:8983/solr/catalog-alma-staging-rebuild bundle exec rake scsb:full`
+`$ SET_URL=http://lib-solr8-prod.princeton.edu:8983/solr/catalog-alma-staging-rebuild bundle exec rake scsb:full`
 
 Indexing jobs for each DumpFile in the dump will be run in the background. To watch the progress of the index, you can login and go to /sidekiq.
 
 This will also index any incremental files we have that were generated after the full dump files.
 
 ### Index Theses
 
-SSH to the bibdata machine that is used for indexing (bibdata-alma-staging) and start a tmux session
+SSH to the bibdata worker machine that is used for indexing and start a tmux session
 
 as deploy user, in `/opt/marc_liberaton/current`
 
 `$ FILEPATH=/home/deploy/theses.json bin/rake orangetheses:cache_theses`
 
 This step takes around 10 minutes. It will create a `theses.json` file in `home/deploy`. Post the file with:
 
-`curl 'http://lib-solr8-staging.princeton.edu:8983/solr/catalog-alma-staging/update?commit=true' --data-binary @/home/deploy/theses.json -H 'Content-type:application/json'`
+`curl 'http://lib-solr8-prod.princeton.edu:8983/solr/catalog-alma-rebuild/update?commit=true' --data-binary @/home/deploy/theses.json -H 'Content-type:application/json'`
 
 ### Index Numismatic Coins
 
+Before running this task, turn off the sneakers workers on production so that any updates that come through after the full index has been generated will be included in the new index and not lost when you swap out the old index.
+
+To turn off sneakers workers:
+- cd in your local princeton_ansible directory -> pipenv shell -> ansible orangelight_alma_prod -u pulsys -m shell -a "sudo service orangelight-sneakers stop"
+
+To index the coins:
+
 - as deploy user, in `/opt/marc_liberaton/current`
-- `$ SET_URL=http://lib-solr8-staging.princeton.edu:8983/solr/catalog-alma-staging-rebuild bundle exec rake numismatics:index:full 2> /tmp/numismatics_index_[date].log`
+- `$ SET_URL=http://lib-solr8-prod.princeton.edu:8983/solr/catalog-alma-rebuild bundle exec rake numismatics:index:full 2> /tmp/numismatics_index_[date].log`
 - It will show a progress bar as it runs. Takes 30 min to an hour.
 - Note that the default log writes to STDERR to distinguish its output from the progress bar
+
+### Logs
+
+Many of these tasks output log messages.
+
+Because traject does not log to datadog we lose log output for indexing that runs in the background. See https://github.com/pulibrary/bibdata#1507
```

Comment:
The link doesn't work for me, so I think it should be https://github.com/pulibrary/bibdata/issues/1507